src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -179,6 +179,150 @@ private UserTypeTypeParametersFormatDefinition()
             }
         }
         #endregion
+        #region User Types - Fields
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.FieldName)]
+        [Name(ClassificationTypeNames.FieldName)]
+        [Order(After = LanguagePriority.NaturalLanguage, Before = LanguagePriority.FormalLanguage)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class UserTypesFieldsFormatDefinition : ClassificationFormatDefinition
+        {
+            private UserTypesFieldsFormatDefinition()
+            {
+                this.DisplayName = EditorFeaturesResources.User_Types_Fields;
```

Comment:
This should probably be User_Members_Fields as Fields are not Types.

Reply:
Hrmm.. although, then these would be sorted separately in teh fonts/colors dialog.  So probably best to leave like this.

Reply:
Y that's what I thought - User_Types is a bit unfortunate but fits 80 - however type in that case is no type in a programming language